src/container/TasksList.js

```diff
@@ -3,26 +3,21 @@
  */
 import React from 'react';
 import TasksItem from './TasksItem';
+import PropTypes from 'prop-types';
 
 export default class TasksList extends React.Component {
-
+    static propTypes = {
+        itemsData: PropTypes.array.isRequired
```

Comment:
Fix prop types, you forget about itemsDelete prop. itemsDelete is actually an example of bad naming too. It's could be better as "onItemDelete" or "onItemRemove"